docs/integration-tests.md

```diff
@@ -22,8 +22,8 @@ Install the "ASP.NET and web development" workload in the "Web & Cloud" category
 
 Depending on which version of Visual Studio you are using, you may have to install some or all of the following:
 
-* .NET 6
-* .NET 7
+* .NET 8
+* .NET 8
```

Comment:
Did you mean .NET 9 here or is this just a duplicate line to be removed?
```suggestion
* .NET 9
```

Reply:
Good catch, thanks.